pkg/plugins/manager.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugins
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	// DefaultPluginPrefix is the default prefix for the plugin binary name.
+	DefaultPluginPrefix = "gtctl-"
+
+	// PluginSearchPathsEnvKey is the environment variable key for the plugin search paths.
+	PluginSearchPathsEnvKey = "GTCTL_PLUGIN_PATHS"
```

Comment:
How about using `PATH` directly?

Reply:
I think we can use `PATH` as the default configuration and add the current directory. Use the diffrent env key can aviod the conflicts.

Reply:
sounds good to me